app/templates/PACKAGE/_DEPENDENCIES.json

```diff
@@ -1,53 +1,53 @@
 {
   "dependencies": {
-    "@fancyapps/fancybox": "latest",
+    "@fancyapps/fancybox": "3.5.3",
```

Comment:
I locked the version due to the latest changes in fancyBox. Please refer to https://github.com/fancyapps/fancybox/issues/2153#issuecomment-447798941.